common/calculate-cpmm.ts

```diff
@@ -260,27 +260,34 @@ export function addCpmmLiquidity(
   return { newPool, liquidity, newP }
 }
 
+const calculateContractLiquidity = (p: number) => (l: LiquidityProvision) => {
+  const oldLiquidity = getCpmmLiquidity(l.pool, p)
+
+  const newPool = addObjects(l.pool, { YES: l.amount, NO: l.amount })
+  const newLiquidity = getCpmmLiquidity(newPool, p)
+
+  const liquidity = newLiquidity - oldLiquidity
+  return liquidity
+}
+
 export function getCpmmLiquidityPoolWeights(
   contract: CPMMContract,
-  liquidities: LiquidityProvision[]
+  liquidities: LiquidityProvision[],
+  excludeAntes = true
```

Comment:
Might be clearer to just remove this param, and always exclude antes, if that's how we use it. Makes the code simpler. Can change later to support including antes if necessary.

Reply:
ok